WordPressVIPMinimum/Sniffs/AbstractVariableRestrictionsSniff.php

```diff
@@ -9,7 +9,7 @@
 
 namespace WordPressVIPMinimum\Sniffs;
 
-use WordPressVIPMinimum\Sniffs\Sniff;
+use PHPCSUtils\Utils\MessageHelper;
```

Comment:
This change is in the wrong commit (in "Add Sniff::merge_custom_array()", while it belongs with "Use PHPCSUtils MessageHelper::addMessage").

Reply:
The "Add Sniff::merge_custom_array()" commit was removed (to be handled by #744), but this line 12 change was added to db681ccf2d61dcac745c510c27ceff532f91b6db.